issuer-web/src/router/index.ts

```diff
@@ -21,7 +23,7 @@ function router(config: AppConfig): VueRouter {
       meta: {
         isPublic: true
       },
-      beforeEnter: validToken
+      beforeEnter: multiGuard([validToken, hasPresReq])
```

Comment:
The landing page should be accessible even without authentication: I am unable to hit it when the `hasPresReq` guard is enabled, we should remove it from here so that the behaviour of the webapp will not change.